test/helpers/tokens.js

```diff
@@ -30,6 +30,14 @@ const getErc20TokenInstances = async (params, from) => {
     })
   );
 
+  if (tokenList.length == 2) {
+    if (tokenList[0].address > tokenList[1].address) {
+      const temp = tokenList[0];
+      tokenList[0] = tokenList[1];
+      tokenList[1] = temp;
+    }
+  }
```

Comment:
What's this doing?

Reply:
Balancer wants to have the tokenlist sorted to address. This function does that